Framework/Kernel/src/ConfigService.cpp

```diff
@@ -86,15 +86,15 @@ namespace { // anonymous namespace for some utility functions
 Logger g_log("ConfigService");
 
 /**
- * Split the supplied string on semicolons.
+ * Split the supplied string on semicolons and commas.
  *
  * @param path The path to split.
  * @returns vector containing the split path.
  */
 std::vector<std::string> splitPath(const std::string &path) {
   std::vector<std::string> splitted;
 
-  if (path.find(';') == std::string::npos) { // don't bother tokenizing
+  if (path.find_first_of(";,") == std::string::npos) { // don't bother tokenizing
```

Comment:
These two needing to be in sync has the potential to cause issues down the line. Can we make a constant called `PATH_DELIMITERS` or something that can be used in both places so this is less likely to happen again?